src/script/main/app.ts

```diff
@@ -891,32 +921,6 @@ class App {
   // Debugging
   //##############################################################################
 
-  /**
-   * Disable debugging on any environment.
-   * @returns {undefined} No return value
-   */
-  disableDebugging() {
-    Config.getConfig().LOGGER.OPTIONS.domains['app.wire.com'] = () => 0;
-    this.repository.properties.savePreference(PROPERTIES_TYPE.ENABLE_DEBUGGING, false);
-  }
-
-  /**
-   * Enable debugging on any environment.
-   * @returns {undefined} No return value
-   */
-  enableDebugging() {
-    Config.getConfig().LOGGER.OPTIONS.domains['app.wire.com'] = () => 300;
-    this.repository.properties.savePreference(PROPERTIES_TYPE.ENABLE_DEBUGGING, true);
-  }
-
-  /**
-   * Report call telemetry to Raygun for analysis.
-   * @returns {undefined} No return value
-   */
-  reportCall() {
-    this.repository.calling.reportCall();
-  }
-
```

Comment:
What about this removed code? Don't we need it anymore?

Reply:
this is not used anywhere at all, calling repo has no method called `reportCall`